### PR TITLE
feat(website): redesign

### PR DESCRIPTION
**Pull request by noahlitvin**

### CODE REVIEW

File: packages/cli/src/chains.ts [hunk 1]
```diff
@@ -1,18 +1,7 @@
...
-  nativeCurrency: {
-    name: 'Ether',
-    symbol: 'ETH',
-    decimals: 18,
-  },
-  rpcUrls: { default: { http: ['http://localhost:8545'] } },
-};
-
-export const chains: viem.Chain[] = [cannonChain, ...Object.values(viemChains)];
+export const chains: viem.Chain[] = [...Object.values(viemChains)];
```
Comment (mjlescano) [inline packages/cli/src/chains.ts:4]: Why this change?

Comment (noahlitvin) [reply packages/cli/src/chains.ts:4]: They added cannon to the viem list so it's redundant now
